.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+docker-buildx.sh
```

Comment:
We don't have a `docker-buildx.sh` just yet so maybe we can add this a bit later